pkg/proxy/accesslog/record.go

```diff
@@ -195,3 +195,22 @@ type LogRecordHTTP struct {
 	// Header is the HTTP header in use
 	Header http.Header
 }
+
+
+
+// LogRecordKafka contains the kafka specific portion of a log record
+// Reference: https://kafka.apache.org/protocol#protocol_api_keys
+type LogRecordKafka struct {
+	// Version of the kafka api used
+	version int
+
+	// Kafka ApiKey -> produce/fetch/createTopic/deleteTopic
```

Comment:
Add a link to the list of possible values for this field. https://kafka.apache.org/protocol#protocol_api_keys

Reply:
sure. I put it at the top of the structure. let me move it down.